src/components/Button/Button.module.scss

```
@@ -0,0 +1,38 @@
+.base__button {
+  display: flex;
+  gap: 9px;
+  position: relative;
+  padding-left: 14px;
+  padding-right: 14px;
+  padding-top: 8px;
+  padding-bottom: 8px;

```

```suggestion
  padding: 8px 14px;
```

некрит, но глаз цепляется